alldogs.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog - Seattle's premiere dog adoption agency</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id='page-header'>
```

Comment:
Be careful with id's. Generally you want to keep those for use with javascript. It's perfectly acceptable to use a class in this case, even if it's for a single use.

Reply:
Haha. Okay. Drupal habit. They slap an id on EVERYTHING!

Reply:
Ha!

Did I mention to not use single quotes on things for your attributes in HTML? If not, make sure to use only double quote marks for HTML.

<header id="page-header">